opentelemetry-sdk/src/error.rs

```diff
@@ -41,5 +41,30 @@ pub enum ShutdownError {
     InternalFailure(String),
 }
 
+#[derive(Error, Debug)]
+/// Errors that can occur during export.
+/// TODO: This should be ExportError, but that can be done after rest of repo changes are done.
+pub enum ExportErrorMetric {
```

Comment:
After checking more, this enum would also need AlreadyShutDown variant. Which means we should be able to re-use the same enum for Export,Shutdown,ForceFlush.

@scottgerring @lalitb Let me know if you agree. I'll make a small PR, just adding the common enum first, and then divide&conquer using it in all places.

Reply:
> I'll make a small PR, just adding the common enum first, and then divide&conquer using it in all places.

Looks good to me.